ibis/mapd/operations.py

```diff
@@ -342,7 +342,7 @@ def literal(translator, expr):
                 msg = 'Unsupported subsecond accuracy {}'
                 raise ValueError(msg.format(value))
             value = value.strftime('%Y-%m-%d %H:%M:%S')
-        return "toDateTime('{0!s}')".format(value)
+        return "DATE '{0!s}'".format(value)
```

Comment:
This should be wrapped in parentheses to make sure the order of operations is what we expect.